<ui version="4.0" >
 <class>TurbulenceForm</class>
 <widget class="QWidget" name="TurbulenceForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxModel" >
     <property name="title" >
      <string>Turbulence model</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QFrame" name="frame" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <layout class="QHBoxLayout" >
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxTurbModel" >
               <property name="minimumSize" >
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip" >
                <string>Code_Saturne key word: ITURB</string>
               </property>
               <property name="statusTip" >
                <string>For mixing length contact the development team.</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameAdvanced" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <layout class="QHBoxLayout" >
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>20</width>
                 <height>21</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelAdvanced" >
               <property name="text" >
                <string>Advanced options</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonAdvanced" >
               <property name="text" >
                <string/>
               </property>
               <property name="icon" >
                <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/22x22/preferences-system.png</iconset>
               </property>
               <property name="iconSize" >
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>31</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameLength" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <layout class="QHBoxLayout" >
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>20</width>
                 <height>21</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelLength" >
               <property name="toolTip" >
                <string>Code_Saturne key word: XLOMLG</string>
               </property>
               <property name="statusTip" >
                <string>The mixing length is a > 0 real.</string>
               </property>
               <property name="text" >
                <string>Length scale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditLength" >
               <property name="maximumSize" >
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip" >
                <string>Code_Saturne key word: XLOMLG</string>
               </property>
               <property name="statusTip" >
                <string>The mixing length is a > 0 real.</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc" />
 </resources>
 <connections/>
</ui>
